packages/editor/src/components/post-author/check.js

```diff
@@ -40,7 +40,10 @@ export default compose( [
 				false
 			),
 			postType: select( 'core/editor' ).getCurrentPostType(),
-			authors: select( 'core' ).getAuthors(),
+			authors: select( 'core' ).getUsers( {
+				who: 'authors',
+				per_page: 2,
+			} ),
```

Comment:
The problem with this change is that it's a different REST API request while `getAuthors` is shared with the other component, which means in one case, we'd do 1 request and the other, we'd do 2. So maybe we should just keep it as is.

Reply:
> while getAuthors is shared with the other component

Out of curiosity, do you know when two requests are the same, is the REST request still made twice?

Reply:
>  Out of curiosity, do you know when two requests are the same, is the REST request still made twice?

If it's the same "selector" call with the same arguments, it's made once.

Reply:
Reverted changes to packages/editor/src/components/post-author/check.js.